Exception when generating JUnit report from MSBuild

This occurs when running any unit test project from MSBuild, with the JUnit attribute (in the .csproj) set to a valid path, and no other reporting attributes set.

## Version
```
xunit 2.4.1 net472
xunit.runner.msbuild 2.4.1 net472
```

## Stack trace
```
error MSB4018: The "xunit" task failed unexpectedly.
System.NullReferenceException: Object reference not set to an instance of an object.
   at Xunit.Runner.MSBuild.CrossPlatform.Transform(IRunnerLogger logger, String outputDisplayName, String resourceName, XNode xml, ITaskItem outputFile)
   at Xunit.Runner.MSBuild.xunit.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()
```

--- a/src/xunit.runner.msbuild/xunit.cs
+++ b/src/xunit.runner.msbuild/xunit.cs
@@ -243,7 +243,7 @@ namespace Xunit.Runner.MSBuild
                     CrossPlatform.Transform(logger, "NUnit", "NUnitXml.xslt", assembliesElement, NUnit);
 
                 if (JUnit != null)
-                    CrossPlatform.Transform(logger, "JUnit", "JUnitXml.xslt", assembliesElement, NUnit);
+                    CrossPlatform.Transform(logger, "JUnit", "JUnitXml.xslt", assembliesElement, JUnit);
             }
 
             // ExitCode is set to 1 for test failures and -1 for Exceptions.

